requirements.txt

```diff
@@ -12,3 +12,5 @@ stable_ts_whisperless==2.17.3
 ruff==0.1.15
 numpy<2
 ctranslate2==4.4.0
+fastapi==0.115.4
+python-multipart==0.0.17
```

Comment:
Where is this used?

Reply:
These are usually required by one of the other imports, upon installation they say, this package also needs xyz installed without installing it as a dependancy itself so I included it

Reply:
Okay if you want to maybe validate that our use case really needs it. The only other occurence of multipart was already in the code. 

If you dont want to maybe at that as a note to the requirements.txt or the readme

Reply:
validated, are needed